src/_common/fireside/post/post-model.ts

```diff
@@ -58,6 +100,7 @@ export class FiresidePost extends Model implements ContentContainerModel {
 	url!: string;
 	view_count?: number;
 	expand_count?: number;
+	is_pinned = false; // Field is only gathered, cannot be directly set.
```

Comment:
is_pinned is always returned, this should be `is_pinned!: boolean`.

Reply:
I did this for backwards comp.